jib-core/src/test/java/com/google/cloud/tools/jib/cache/CacheMetadataTest.java

```diff
@@ -177,7 +187,8 @@ public void testFilter_byNoEntries() throws CacheMetadataCorruptedException {
 
     LayerEntry fakeLayerEntry =
         new LayerEntry(
-            Paths.get("some/source/file", "some/source/directory"), Paths.get("extractionPath"));
+            Paths.get("some/source/file", "some/source/directory"),
```

Comment:
Unrelated, but this example seems weird, which just results in a path of `/some/source/file/some/source/directory`. Maybe the intention in the past was to pass two `Path` as a list.